app/assets/stylesheets/components/editor-toolbar.scss

```diff
@@ -1,4 +1,24 @@
 .editor-toolbar {
   display: flex;
   background: var(--base-0);
+
+  .toolbar-btn .spinner-or-cancel {
```

Comment:
This new css controls the image upload "cancel" button - which displays a spinner unless you hover or focus the button (in which case you see the cancel icon)

I looked at tailwind-y ways to achieve this but I don't think we currently use e.g. `hover:opacity-0`